os/include/sys/syscall.h

```diff
@@ -143,17 +143,18 @@
 #endif
 
 #define SYS_task_delete                __SYS_task_delete
-#define SYS_task_restart               (__SYS_task_delete + 1)
+#define SYS_task_event_wait            (__SYS_task_delete + 1)
```

Comment:
Do you have any reason why you add it at middle of the list instead of end of the list?

Reply:
For gathering task_xxx APIs.